<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Progam Antrian Bank Sederhana</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Antrian Saat Ini</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>582</width>
            <height>201</height>
           </size>
          </property>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>178</width>
             <height>172</height>
            </rect>
           </property>
           <property name="title">
            <string>CS-1</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLCDNumber" name="lcdNumberCs1">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>71</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>71</height>
               </size>
              </property>
              <property name="digitCount">
               <number>3</number>
              </property>
              <property name="intValue" stdset="0">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelNameCs1">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="pushButtonIstirahatCs1">
                <property name="text">
                 <string>Istirahat</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonLanjutCs1">
                <property name="text">
                 <string>Selanjutnya</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>20</y>
             <width>178</width>
             <height>172</height>
            </rect>
           </property>
           <property name="title">
            <string>CS-2</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLCDNumber" name="lcdNumberCs2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>71</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>71</height>
               </size>
              </property>
              <property name="digitCount">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelNameCs2">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="pushButtonIstirahatCs2">
                <property name="text">
                 <string>Istirahat</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonLanjutCs2">
                <property name="text">
                 <string>Selanjutnya</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>20</y>
             <width>178</width>
             <height>172</height>
            </rect>
           </property>
           <property name="title">
            <string>CS-3</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLCDNumber" name="lcdNumberCs3">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>71</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>71</height>
               </size>
              </property>
              <property name="digitCount">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelNameCs3">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButtonIstirahatCs3">
                <property name="text">
                 <string>Istirahat</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonLanjutCs3">
                <property name="text">
                 <string>Selanjutnya</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_6">
          <property name="minimumSize">
           <size>
            <width>582</width>
            <height>201</height>
           </size>
          </property>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>178</width>
             <height>172</height>
            </rect>
           </property>
           <property name="title">
            <string>TELLER-1</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLCDNumber" name="lcdNumberTeller1">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>71</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>71</height>
               </size>
              </property>
              <property name="digitCount">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelNameTeller1">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="pushButtonIstirahatTeller1">
                <property name="text">
                 <string>Istirahat</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonLanjutTeller1">
                <property name="text">
                 <string>Selanjutnya</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>20</y>
             <width>178</width>
             <height>172</height>
            </rect>
           </property>
           <property name="title">
            <string>TELLER-2</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLCDNumber" name="lcdNumberTeller2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>71</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>71</height>
               </size>
              </property>
              <property name="digitCount">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelNameTeller2">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="pushButtonIstirahatTeller2">
                <property name="text">
                 <string>Istirahat</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonLanjutTeller2">
                <property name="text">
                 <string>Selanjutnya</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Daftar</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>CS</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QFrame" name="frame">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>43</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QSpinBox" name="spinBoxCs">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditCs"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonAntriCs">
                <property name="text">
                 <string>Antri</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidgetCs">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustIgnored</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>TELLER</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>43</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QSpinBox" name="spinBoxTeller">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditTeller"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonAntriTeller">
                <property name="text">
                 <string>Antri</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidgetTeller">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustIgnored</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Info</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>241</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;Nama : Arinadi Nur Rohmad&lt;br /&gt;NIM : 191110080&lt;br /&gt;Email : arinadi.nur@gmail.com&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;HP : 081287734946&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>271</width>
          <height>131</height>
         </rect>
        </property>
        <property name="title">
         <string>Total Antrian</string>
        </property>
        <widget class="QLCDNumber" name="lcdNumberTotalAntrian">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>251</width>
           <height>101</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
